fastlane/lib/fastlane/actions/git_commit.rb

```diff
@@ -10,8 +10,17 @@ def self.run(params)
 
         skip_git_hooks = params[:skip_git_hooks] ? '--no-verify' : ''
 
-        result = Actions.sh("git commit -m #{params[:message].shellescape} #{paths} #{skip_git_hooks}".strip)
-        UI.success("Successfully committed \"#{params[:path]}\" 💾.")
+        command = "git commit -m #{params[:message].shellescape} #{paths} #{skip_git_hooks}".strip
```

Comment:
We could probably move this line after the `if params[:allow_nothing_to_commit]` block since this line isn't needed until after that 😊